execute/wait.go

```diff
@@ -51,7 +51,7 @@ func Wait(hosts []string, timeout time.Duration) {
 	wg.Wait()
 
 	if !timer.Stop() {
-		log.Fatalf("Error: One or more services timed out after %d second(s)", timeout)
+		log.Fatalf("Error: One or more services timed out after %v", timeout)
```

Comment:
Yeah, `time.Duration`'s default formatting includes the unit: https://play.golang.org/p/49k0GBiVS-